daffodil-japi/src/test/java/org/apache/daffodil/example/TestJavaAPI.java

```diff
@@ -122,7 +122,7 @@ public void testJavaAPI1() throws IOException, ClassNotFoundException {
         DebuggerRunnerForJAPITest debugger = new DebuggerRunnerForJAPITest();
 
         org.apache.daffodil.japi.Compiler c = Daffodil.compiler();
-        java.io.File schemaFile = getResource("/test/japi/mySchema1.dfdl.xsd");
+        java.io.File schemaFile = new File("daffodil-japi/src/test/resources/test/japi/mySchema1.dfdl.xsd");
```

Comment:
What is the reason for replacing getResource (was it a function we defined ourselves?  I thought Java's getResource returned an URI or null, not a File) with new File?